dev/fattest.simplicity/src/componenttest/custom/junit/runner/FATRunner.java

```diff
@@ -645,6 +652,13 @@ private String getFFDCHeader(RemoteFile ffdcLogFile) throws Exception {
                 annotationListPerClass.add(exceptionClass);
             }
         }
+        AllowedFFDC servletFFDC = m.getMethod().getDeclaringClass().getAnnotation(AllowedFFDC.class);
+        if (servletFFDC != null) {
```

Comment:
in cases where `@TestServlet` is not used, `getTestClass().getJavaClass()` will be == to `m.getMethod().getDeclaringClass()`, I think we should check for that case to avoid double-adding the allowed FFDC classes